ui/pages/confirmations/confirmation/confirmation.js

```diff
@@ -375,34 +373,28 @@ export default function ConfirmationPage({
   useEffect(() => {
     async function fetchSafeChainsList(_pendingConfirmation) {
       try {
-        if (useSafeChainsListValidation) {
```

Comment:
Same concept as the comment in `ui/hooks/useIsOriginalNativeTokenSymbol.js`. We don't need to check `useSafeChainsListValidation`, as it is done in `getWellknownChains`. And the setting does NOT determine if we can or cannot use a list of chains... only that we cannot contact a 3rd party network.